docs/DeveloperGuide.md

```diff
@@ -451,52 +463,51 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
       Use case resumes at step 2.
 
 
-**Use case: Deleting a single resident**
+**Use case: UC 7. Deleting a single resident**
```

Comment:
Space between '**UC**' and '**7**' here.

Reply:
Hmm, from here on, it seems there are more of the spaced ones as compared to the ones without spacing. Just try to standardize it, doesn't matter which version you pick, **spaced** or **no spaced**.